omniflixhub/assetlist.json

```diff
@@ -39,6 +39,37 @@
       },
       "type_asset": "sdk.coin"
     },
+    {
+      "description": "GATA Yield DAO Governance Token powered by OmniFlixHub",
+      "denom_units": [
+        {
+          "denom": "factory/omniflix1fwphj5p6qd8gtkehkzfgac38eur4uqzgz97uwvf6hsc0vjl004gqfj0xnv/ygata",
+          "exponent": 0
+        },
+        {
+          "denom": "ygata",
+          "exponent": 6
+        }
+      ],
+      "base": "factory/omniflix1fwphj5p6qd8gtkehkzfgac38eur4uqzgz97uwvf6hsc0vjl004gqfj0xnv/ygata",
+      "name": "Yield GATA",
+      "display": "yGATA",
```

Comment:
"Display denomination yGATA is not defined as a denom_unit"
the uni defined is "ygata".Changing this value to "ygata" (alllower-case) will resolve this issue.